packages/core/ios/RNEmbrace/EmbraceManager.m

```diff
@@ -174,6 +174,10 @@ @interface RCT_EXTERN_MODULE(EmbraceManager, NSObject)
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 
+RCT_EXTERN_METHOD(appendCrashInfo:(NSString *)key
```

Comment:
is this something we need to expose to the JS layer? It looked liked everything was being done in `logUnhandledJSException` ?

Reply:
not really, I just thought we wanted to expose this one too since it exists, but I can remove this method if we don't want it